### PR TITLE
Some doc cleanup of Triangulation

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

File: lib/matplotlib/tri/triangulation.py [hunk 1]
```diff
@@ -76,29 +80,32 @@ def __init__(self, x, y, triangles=None, mask=None):
     def calculate_plane_coefficients(self, z):
         """
         Calculate plane equation coefficients for all unmasked triangles from
-        the point (x,y) coordinates and specified z-array of shape (npoints).
-        Returned array has shape (npoints,3) and allows z-value at (x,y)
+        the point (x, y) coordinates and specified z-array of shape (npoints).
+        The returned array has shape (npoints, 3) and allows z-value at (x, y)
         position in triangle tri to be calculated using
-        z = array[tri,0]*x + array[tri,1]*y + array[tri,2].
+        ``z = array[tri, 0] *x  + array[tri, 1] * y + array[tri, 2]``.
```
Comment (anntzer) [inline lib/matplotlib/tri/triangulation.py]: space before x

**Comment by anntzer** [comment]
thanks

File: lib/matplotlib/tri/triangulation.py [hunk 2]
```diff
@@ -12,19 +12,23 @@ class Triangulation(object):
...
-    mask : boolean array_like of shape (ntri), optional
+    mask : boolean array-like of shape (ntri), optional
         Which triangles are masked out.
 
     Attributes
     ----------
-    `edges`
-    `neighbors`
+    edges : int array of shape (nedges, 2)
+        See `~.Triangulation.edges`
```
Comment (ImportanceOfBeingErnest) [inline lib/matplotlib/tri/triangulation.py]: This line is throwing an error for me when building the docs locally. 

`d:\...\matplotlib\lib\matplotlib\tri\triangulation.py:docstring of matplotlib.tri.Triangulation:40:
toctree references unknown document 'api/matplotlib.tri.Triangulation.edges'`

I wonder anyways why 
```
   edges : int array of shape (nedges, 2)
        See `~.Triangulation.edges`
```
translates into 

image

see [link to built doc](https://11142-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/tri_api.html)

Is there some novel doc interpolation going on? Does this have any additional requirement?

Comment (tacaswell) [reply lib/matplotlib/tri/triangulation.py]: The built documentation seems to be pulled from the docstring on the `edges` property below.  The issues maybe different versions of numpydoc behaving differently...

Comment (ImportanceOfBeingErnest) [reply lib/matplotlib/tri/triangulation.py]: Good catch, thanks for the hint. I update the requirement in https://github.com/matplotlib/matplotlib/pull/11667

**Pull request merged**